### PR TITLE
Add sha256 files for the Mac and Linux installers.

### PR DESCRIPTION
**Pull request by dellis1972**
Fix up the android build to it will work with the latest SDK on Mac.

### CODE REVIEW

**Comment by tomspilman** [comment]
@dellis1972 - Is this related to https://github.com/MonoGame/MonoGame/issues/5929 ?

**Comment by dellis1972** [comment]
yes

**Comment by dellis1972** [comment]
@tomspilman this will produce

On Windows
`Installers/Windows/MonoGameSetup.exe.sha`

On Mac
`Installers/MonoGame.pkg.sha`
`Installers/Pipeline.MacOS.pkg.sha`
`Installers/monogame-sdk.run.sha`

File: Installers/default.build [hunk 1]
```diff
@@ -34,6 +34,17 @@
         <exec program="makensis" workingdir="Windows" basedir="C:\Program Files (x86)\NSIS">
 		      <arg value="MonoGame.nsi"/>
         </exec>
+        <echo append="false" file="Windows/hash.bat">
+@ECHO OFF
+FOR /f "tokens=*" %%i IN ('@certutil -hashfile %1 %2 ^| find /v "hash of file" ^| find /v "CertUtil"') DO SET r=%%i
+SET r=%r: =%
+ECHO %r%  %1 > %3
+        </echo>
```
Comment (tomspilman) [inline Installers/default.build]: Seems like a bit of a dirty trick to do this here.  Why not just create a bat file and add it to the repo?

**Comment by tomspilman** [comment]
@dellis1972 - Are these SHA files still useful once we start signing the Windows installer?

I guess i just post the link to the SHA along side the installer?

File: Installers/default.build (hunk 1 above)
Comment (dellis1972) [reply Installers/default.build]: I can no problem.

**Comment by dellis1972** [comment]
Once we start signing the windows installer probably not (on windows). We should also sign the Mac one too if we can setup an apple account.

Yup we can publish the .sha link next to the link for the installers.

**Pull request closed without merging**